hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/buffer/BackupMessagesLoader.java

```diff
@@ -141,6 +160,30 @@ private boolean sendMessageIfNeeded(Message message, Optional<Topic> topic, Stri
         }
     }
 
+    private void waitOnBrokerTopicMetadata(Topic topic) {
```

Comment:
i think we could at least add some log messages like "Topic {} is not available, checked {} times" i "Topic loaded"? this only happens on startup after all and might give us some hint of what is going on in case we see frontend hanging